packages/owl-bot/cloud-build/update-pr.yaml

```diff
@@ -97,6 +97,29 @@ steps:
       - 'GITHUB_TOKEN=$_GITHUB_TOKEN'
       - 'HOME=/workspace'
 
+  # TODO: remove when we find a better resolution.
```

Comment:
I think we need a 
```
env:
    - 'HOME=/workspace'
```

On each of these new steps.

Reply:
Fixed